content/docs/contributing/building.md

```diff
@@ -3,123 +3,211 @@ title: Building cert-manager
-> a [clean workaround might be available](https://groups.google.com/g/bazel-discuss/c/13uPDObyfQg/m/UjPbalztCQAJ).
+## Prerequisites
 
-## A quick intro to Bazel
+There are very few other requirements needed for developing cert-manager, and crucially the build system should tell you with a friendly error
+message if there's anything missing. If you think an error message which relates to a missing dependency is unhelpful, we consider that a bug and
+we'd appreciate if you raised [an issue](https://github.com/cert-manager/cert-manager/issues/new?assignees=&labels=&template=bug.md) to tell us about it!
 
-Bazel has 3 main commands which we use:
+You should install the following tools before you start developing cert-manager:
 
-`bazel build [...]` will build and compile code for you e.g. `bazel build //cmd/ctl` will build our CLI. 
-`bazel test [...]` will run any tests for a given package
-`bazel run [...]` is only used to run certain scripts not the compiled code (unlike Go). e.g. `bazel run //hack/bin:helm` will download and run Helm.
+- [git](https://git-scm.com/)
+- [curl](https://curl.se/)
+- GNU Coreutils (available via [homebrew](https://formulae.brew.sh/formula/coreutils) for macOS)
+- `jq` (available in package managers and in homebrew)
+- `docker` (or `podman`, see [Container Engines](#container-engines) below)
 
-### Package format
+## Getting Started
 
-After any Bazel command you will see something that looks like a path.
-Let's take `bazel run //hack/bin:helm` as an example:
+The vast majority of commands which you're likely to need to use are documented via `make help`. That's probably the first place to start if you're
+developing cert-manager. We'll also provide an overview on this page of some of the key targets and things to bear in mind.
 
-* `//` is the cert-manager project root, no matter in which directory under cert-manager you are it will find it
-* `hack/bin` is the path where the code is to execute/build/test you will for example see `pkg/acme/` to run ACME tests
-* `:helm` is the part of the Bazel file to execute, these are defined in the Bazel config themselves.
+### Go Versions
 
-> **TIP**: `...` is a recursive lookup in Bazel, it will run al tests in all subfolders when set, it is also the easiest way to invoke them.
-> For example `bazel test //pkg/...` will test all tests in all packages.
+cert-manager can be built with the version of Go you've installed locally on your system. Alternatively, you can "vendor" Go to ensure you use the
+same version that's built with in CI.
 
-## Help so much Bazel!
+To check which version of Go is _currently_ being used, run: `make which-go`.
 
-No worries we have a lot of helper scripts for you!
-Need to set up a local cluster and install cert-manager in it? Take a look at [our kind documentation](./kind.md).
+To start using a vendored Go, run: `make vendor-go`. You only need to run `vendor-go` once and it'll be "sticky", being used for all future make invocations
```

Comment:
This seems a little confusing tbh, how would I remember if I ran this comand already and how can I be sure that it's really 'unvendored'? But I guess this is more a question of whether we want to do it this way not a documentation issue.

Suggestion: maybe state clearly that by default system's Go is used and add all the `vendor/unvendor` as optional steps so this is more straightforward to read?

Reply:
I'll try rewording this to make it clearer!

I added `make which-go` to address this point of possible confusion specifically, and that's documented just above here; I'll add an example showing how I expected vendoring and unvendoring to work!